certs/certs.go

```diff
@@ -30,3 +39,46 @@ type Repository interface {
 	// RetrieveBySerial retrieves a certificate for a given serial ID
 	RetrieveBySerial(ctx context.Context, ownerID, serialID string) (Cert, error)
 }
+
+func LoadCertificates(caPath, caKeyPath string) (tls.Certificate, *x509.Certificate, error) {
+	var tlsCert tls.Certificate
```

Comment:
They are used before `LoadX509KeyPair`. When `caPath` and `caKeyPath` is `==` `""`

Reply:
you can return an empty struct and nill, for that case

Reply:
at `b, err := ioutil.ReadFile(caPath)` on error handling should we return an empty `tlsCert` yet is was created or not?

Reply:
I also prefer Sammy's approach. I find explicit `return tls.Certificate{}, nil, nil` in case of error just as fine and a bit more precise and efficient (when reading `return tlsCert, caCert, nil` you do not know what are actual values; but you also do not care). This is a matter of personal preference.